### ISSUE TITLE
Toxics are not set when creating a proxy for the first time.

### ISSUE THREAD

**Issue by jpittis** [Shopify/toxiproxy#79]
When using the Go client as follows, toxics are not added to a proxy.
Given `var p []toxiproxy.Proxy` with toxics.
```` Go
proxy := client.NewProxy(&p)
proxy.Create()
````
This does not properly add the configured toxics from `p` onto the proxy.

They must be added after the fact using `Proxy.SetToxic`.

**Comment by Sirupsen** [comment]
yeah this is a known "issue". @xthexder I don't think this is any different in 2.0?

**Comment by xthexder** [comment]
Yeah, it's not any different currently, just a little less misleading. I think it would be reasonable to allow toxics to be specified in the `Populate()` function
